src/tokens/TokenBalances.tsx

```diff
@@ -342,6 +343,8 @@ function TokenBalancesScreen({ navigation, route }: Props) {
         contentContainerStyle={{
           paddingBottom: insets.bottom,
           opacity: listHeaderHeight > 0 ? 1 : 0,
+          // Allow header scroll even when not enough assets or positions
+          minHeight: variables.height + listHeaderHeight,
```

Comment:
Why do we need header scroll when everything is visible?

Reply:
This was more of a personal preference, but I agree it's not needed. Reverted in https://github.com/valora-inc/wallet/pull/4149/commits/7b161d7c1627b7c60d4f1cc4529dc2e761b2c6a2!